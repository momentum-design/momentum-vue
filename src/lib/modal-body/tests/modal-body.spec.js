import { mount } from '@vue/test-utils';
import ModalBody from '../index.vue';

describe('ModalBody', () => {
  it('should match snapshot', () => {
    const wrapper = mount(ModalBody);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it('should render one ModalBody', () => {
    const wrapper = mount(ModalBody);

    expect(wrapper.findAll('.md-modal__body').length).toEqual(1);
  });

  it('should render children', () => {
    const wrapper = mount(ModalBody, {
      slots: {
        default: '<div class="testingforMB" />'
      }
    });

    expect(wrapper.findAll('.testingforMB').length).toEqual(1);
  });
});

