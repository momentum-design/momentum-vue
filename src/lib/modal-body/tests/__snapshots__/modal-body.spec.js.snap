// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBody should match snapshot 1`] = `<div class="md-modal__body"></div>`;
